vector_double.cpp

```diff
@@ -0,0 +1,207 @@
+#include "vector_double.hpp"
+
+
+template <typename T>
+Vector<T>::Vector()
+{
+    size_v = 0;
+    space  = 0;
+    elem   = nullptr;
+
+}
+
+
+// Copy Constructor
+template<class T>
+Vector<T>::Vector(const Vector<T>& b)
+{
+    space = b.space;
+    size_v = b.size_v;
+    elem = b.elem;
+
+}
+template<class T>
+Vector<T>::Vector(Vector<T>&& obj)
+{
+    space = obj.space;
+    size_v = obj.size_v;
+    elem = obj.elem;
+}
+
+//explicit
+template<class T>
+Vector<T>::Vector(unsigned int)
+{
+    size_v = 0;
+    space  = 0;
+    elem   = nullptr;
+}
+
+
+template<class T>
+Vector<T>::~Vector()
+{
+    delete [] elem;
```

Comment:
syntax nit:
```suggestion
    delete[] elem;
```

Reply:
I would prefer that you put the code as follow:
```cpp
template<typename T>
Vector<T>::~Vector()
{delete[] elem;}
```
but that is just syntax preferences